ministryofjustice/correspondence_tool_staff
# CT-3522 Changed the perf calculation in appeal reports

## Description

This PR is to change the calculation of performance percentage for appeal reports to be in line with other performance reports such as business unit performance report

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3522
### Deployment


### Manual testing instructions

1 - Login in as London users
2 - Click "Reports" from the top menu
3 - Click 'Create report'
4 - Choose FOI / Appeal reports
5 -  Provide the time frame as the input 
6 -  Click 'Create  custom report'
7 - Download it and check the figure under the column of 'performance percentage'
repeat 4 -  7 for SAR case type
